controllers/provider-azure/docs/usage-as-end-user.md

```diff
@@ -64,6 +68,8 @@ Via the `.zoned` boolean you can tell whether you want to use Azure availability
 If you don't use zones then an availability set will be created and only basic load balancers will be used.
 Zoned clusters use standard load balancers.
 
+In the `identity` section you can specify an [Azure user-assigned managed identity](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview#how-does-the-managed-identities-for-azure-resources-work) which should be attached to all cluster worker machines. With `identity.name` you can specify the name of the identity and with `identity.resourceGroupName` you can specify the resource group which contain the identity resource on Azure. Via the `identity.acrAccess` you can configure the worker machines to use the passed identity for pulling from an [Azure Container Registry (ACR)](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-intro).
```

Comment:
Can you explicitly state here that the identity must be created by the end-user itself before passed here (i.e., that the Azure controller does not create it automatically (or does it?))?